timeline.html

```diff
@@ -1 +1,401 @@
+      <!-- Main stuff here -->
+      <div class="container-fluid information-panel margin-at-top">
+
+        <!-- All encompassing row -->
+        <div class="row">
+          <!-- Divide into left and right columns -->
+          <div class="col-md-5"> <!-- Left column -->
+
+            <div class="row box-me margin-me"> <!-- About -->
+              <div class="col-md-12">
+
+                <div class="row box-me"> <!-- Title box -->
+                  <div class="col-md-12" align="center">
+                    <p>About</p>
+                  </div>
+                </div>
+
+                <div class="row"> <!-- Information -->
+                  <div class="col-md-12">
+                    <p>Born on: July 31st, 1980</p>
+                    <p>Went to school at: Hogwarts</p>
+                    <p>Hometown: Godrick's Hollow, England</p>
+                    <p>Currently lives: Godrick's Hollow</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="row box-me margin-me"> <!-- Photos -->
+              <div class="col-md-12">
+
+                <div class="row box-me"> <!-- Title box -->
+                  <div class="col-md-12" align="center">
+                    <p>Photos (123)</p>
+                  </div>
+                </div>
+
+                <div class="row"> <!-- First row of photos -->
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                </div>
+
+                <div class="row"> <!-- Second row of photos -->
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                </div>
+
+                <div class="row"> <!-- Third row of photos -->
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                  <div class="col-md-4">
+                    <img src="assets/icon_photo_small.png">
+                  </div>
+                </div>
+                <div class="row"> <!-- See more photos -->
+                  <div class="col-md-12" align="center">
+                    <a href="hello">See More Photos</a>
+                  </div>
+                </div>
+
```

Comment:
Somehow all of your column boxes are off-center--justified left-- and I can't figure out why!!!